Nanoleaf-wpf/Views/MainWindows/MainWindow.xaml

```diff
@@ -14,28 +14,28 @@
             <Button  Grid.Row="0" VerticalAlignment="Top" Height="35" Background="{x:Null}" BorderBrush="{x:Null}" Click="AddSchedule_Click" Margin="0,51,10,0" HorizontalAlignment="Right" Width="35">
                 <Image Source="../../Icons/appbar.add.png"/>
             </Button>
-            <StackPanel Height="Auto" Margin="10,86,10,0" VerticalAlignment="Stretch" x:Name="ScheduleList">
+            <StackPanel Height="Auto" Margin="10,86,10,104.333" VerticalAlignment="Stretch" x:Name="ScheduleList">
```

Comment:
4 alignments might be wrong and should be fixed with the correct Horizontal Allignment

Reply:
In this case it is correct, since it is located in the middle of the grid, making it scale nicely if the window size is increased.